lbrynet/lbrynet_daemon/Daemon.py

```diff
@@ -1459,19 +1461,22 @@ def jsonrpc_claim_show(self, name=None, txid=None, nout=None, claim_id=None):
     @AuthJSONRPCServer.auth_required
     @defer.inlineCallbacks
     @AuthJSONRPCServer.flags(force='-f')
-    def jsonrpc_resolve(self, uri, force=False):
+    def jsonrpc_resolve(self, force=False, uri=None, uris=[]):
         """
         Resolve a LBRY URI
 
         Usage:
-            resolve <uri> [-f]
+            resolve (<uri> | --uri=<uri>) [<uris>...] [-f]
```

Comment:
Then you also dont need the weird special case of one uri being resolved. You always return a dict of resolved uris, keyed by uri (though this does break BC and I don't see a clean fix for it off the top of my head).